arangod/Cluster/ClusterFeature.cpp

```diff
@@ -685,10 +689,14 @@ void ClusterFeature::stop() {
     return;
   }
 
-  if (!_agencyCallbackRegistry->unregisterCallback(_hotbackupRestoreCallback)) {
-    LOG_TOPIC("84152", INFO, Logger::BACKUP) << "Strange, we could not "
-      "unregister the hotbackup restore callback.";
+#ifdef USE_ENTERPRISE
+  if (_hotbackupRestoreCallback != nullptr) {
+    if (!_agencyCallbackRegistry->unregisterCallback(_hotbackupRestoreCallback)) {
+      LOG_TOPIC("84152", INFO, Logger::BACKUP) << "Strange, we could not "
```

Comment:
Is this even worth logging? The use can nothing do about it and we neither and nobody cares actually.

Reply:
I changed the log level to debug here.